src/components/button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
```

Comment:
**OPTIONAL**
We can write this as
```js 
const Button = { name } => {
...
```
Since the component only accepts `name` as the only prop